common/components/chrome/SubHeader.jsx

```diff
@@ -170,7 +170,7 @@ class SubHeader extends React.Component<{||}, State > {
   };
 
   _closeDropdown(event): void {
-    if (this.anchorEl.contains(event.target)) {
+    if (this.anchorEl && this.anchorEl.contains(event.target)) {
```

Comment:
I added this to the conditional because I was getting an undefined error in the following situation: On mobile, log out and go to the Find Projects page. Open the hamburger menu and click Find Projects (aka the page you're on) - I don't think this is the best way to go about it since I think it triggers a page reload (something that checks if page requested = current page and just closes the slider if true would be ideal?) but it was quick.